kpi/deployment_backends/kobocat_backend.py

```diff
@@ -349,3 +349,13 @@ def get_enketo_survey_links(self):
             except KeyError:
                 pass
         return links
+
+    def _submission_count(self):
+        _deployment_data = self.asset._deployment_data
+        id_string = _deployment_data['backend_response']['id_string']
+        # avoid migrations from being created for kc_reader mocked models
+        # there should be a better way to do this, right?
```

Comment:
KC keeps it updated [here](https://github.com/kobotoolbox/kobocat/blob/master/onadata/apps/logger/models/instance.py#L61). It ran pretty quickly when I tested it. I suspect an index or two were added since `num_of_submissions` was first needed